source/_posts/2017/2017-02-15-minimalistic-way-to-create-your-first-nette-extension.md

```diff
@@ -121,7 +120,7 @@ final class FileSystemExtension extends CompilerExtension
         // this method loads servcies from config and registers them do Nette\DI Container
         Compiler::loadDefinitions(
             $this->getContainerBuilder(),
-            $this->loadFromFile(__DIR__.'/../config/services.neon')['services']
+            $this->loadFromFile(__DIR__.'/../config/services.neon')
```

Comment:
To je dobrý dotaz, to jsem přehlédl a nezkoušel. Pak bych řekl, že to bude asi OK.